### ISSUE TITLE
What does the site hierarchy look like?

### ISSUE THREAD

**Issue by waldoj** [electionlaw/eBenchbook#17]
I suspect that the user begins by selecting a date, and that provides a list of topics. The user selects a topic, and that provides a list of laws. (Or maybe structural units that are drilled through?) Then, upon selecting a law, the site's primary information becomes available—the text of the law, analysis, footnotes, links to supporting information, and perhaps comments.

**Comment by waldoj** [comment]
I think we should tend towards a flatter structure. I don't think it's helpful to nest laws as per the actual structure of the legal code in question, making people drill down through chapters, parts, etc.

**Comment by waldoj** [comment]
OK, so that gives us something like this:

home page ➔ state ➔ topic ➔ law

It's not clear to me whether laws need to be on a different page than topics. That is, maybe each topic is a header, and all relevant laws are listed below that topic. Or maybe that results in an unreasonably long, unwieldy page. I'll need to see what that looks like.

**Comment by waldoj** [comment]
I think I've missed something: there are subtopics. For examples, under "Recounts," topics include "Petition," "Presidential Electors," "Costs," etc. How does this fit into our site structure?

**Comment by waldoj** [comment]
Resolution of this hinges on #18.

**Comment by rgreenwm** [comment]
I agree that we should not nest laws, make user drill down through chapters. Our goal is to make the code come alive and that would seem to have the opposite effect.

**Comment by rgreenwm** [comment]
On the question of subtopics, I'm afraid this is an issue of quality/conformity control. Some students have approached this by asking topical questions (eg, for recounts, timeline, cost, etc.) Others have hewn closer to the code. This is of course our big question. I suppose there's some advantage to having tried it both ways. I need to sit down with it and think about what works best. I'm fearful that the answer might depend on the particular state code, which I gather is a bad answer from the perspective of building this thing...
